docs/modules/ROOT/pages/tmail-backend/features/openpaas-integration.adoc

```diff
@@ -0,0 +1,34 @@
+= OpenPaaS Integration
+:navtitle: OpenPaaS Integration
+
+OpenPaaS, developed by Linagora, is a collaborative platform offering features such as calendar management, contact storage, and more. TMail integrates with OpenPaaS to offer enhanced email and collaboration features.
```

Comment:
Link for OpenPaaS

Reply:
Tmail integrates with OpenPaaS to offer integrations with OpenPaaS Calendar and address book.